src/io/flutter/logging/FlutterConsoleLogManager.java

```diff
@@ -6,62 +6,240 @@
 
 package io.flutter.logging;
 
+
 import com.google.common.annotations.VisibleForTesting;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParser;
-import com.google.gson.JsonSyntaxException;
+import com.google.gson.*;
 import com.intellij.execution.ui.ConsoleView;
 import com.intellij.execution.ui.ConsoleViewContentType;
+import com.intellij.openapi.diagnostic.Logger;
 import com.intellij.openapi.util.text.StringUtil;
+import com.intellij.ui.JBColor;
 import com.intellij.ui.SimpleTextAttributes;
 import com.intellij.util.concurrency.QueueProcessor;
+import io.flutter.inspector.DiagnosticLevel;
+import io.flutter.inspector.DiagnosticsNode;
+import io.flutter.inspector.InspectorService;
+import io.flutter.run.daemon.FlutterApp;
 import io.flutter.vmService.VmServiceConsumers;
 import org.dartlang.vm.service.VmService;
 import org.dartlang.vm.service.consumer.GetObjectConsumer;
 import org.dartlang.vm.service.element.*;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.ArrayList;
 import java.util.Collections;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
 
 /**
  * Handle displaying dart:developer log messages and Flutter.Error messages in the Run and Debug
  * console.
  */
 public class FlutterConsoleLogManager {
-  public static final boolean SHOW_STRUCTURED_ERRORS = false;
+  private static final Logger LOG = Logger.getInstance(FlutterConsoleLogManager.class);
+
+  public static final boolean SHOW_STRUCTURED_ERRORS = true;
```

Comment:
Ah, note that this PR turns error display on. I could also change this to an opt-in user setting.